external/iotivity/iotivity_1.2-rel/extlibs/libcoap/SConscript

```diff
@@ -50,6 +50,15 @@ libcoap_zip_file         = os.path.join(src_dir, 'extlibs', 'libcoap', libcoap_b
 libcoap_url              = libcoap_repo_url + '/archive/' + libcoap_branch + '.zip'
 libcoap_checkout_command = 'git clone ' + libcoap_repo_url + '.git extlibs/libcoap/libcoap --branch ' + libcoap_branch
 
+if GetOption('clean'):
+        lib_path = os.path.join(src_dir, 'extlibs', 'libcoap', 'libcoap.a')
+        print "Library path: %s" %lib_path
```

Comment:
I think if you output the path, it would be better to specify it as the 'Libcoap' library.

Reply:
After `make`, if I run `make clean` command, TizenRT will delete `libcoap.a` file well. However, if I repeat `make clean`, TizenRT does not need to print the path because the `libcoap.a` file does not exist.

Reply:
Thanks for your comment.
Output path was added just for verification. I will remove it and amend.
Moreover, as libcoap library is kept at multiple places, it might not be useful to mention just one of them in the builg log as the path of libcoap library.